src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -189,11 +187,9 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
               } else {
                 // There is no logged in user for a valid, used registration key, redirect to login page
                 // eslint-disable-next-line no-lonely-if
-                if (this.entityType === 'student') {
-                  window.location.href = `${this.backendUrl}${auth.studentLoginUrl}`;
-                } else if (this.entityType === 'instructor') {
-                  window.location.href = `${this.backendUrl}${auth.instructorLoginUrl}`;
-                }
+                this.loadCourseInfo();
+                this.loadPersonName();
+                this.loadFeedbackSession();
```

Comment:
I don't think this block should be changed. Look at the above comment as for why the block acts this way.